client/.env.example

```diff
@@ -0,0 +1,4 @@
+REACT_APP_MAPBOX_TOKEN=
+REACT_APP_DEV_API_URL=
+REACT_APP_DEV_DISABLE_LOGIN=
+REACT_APP_DEV_ACCESS_TOKEN=
```

Comment:
we should probably change these to `REACT_APP_API_URL` and `REACT_APP_ACCESS_TOKEN` since we will be using it in prod as well. dont forget to change usersSlice and API_URL.js